### PR TITLE
Logan/eng 469 fix tinacms dependency vulnerabilities

### PR DESCRIPTION
**Pull request by logan-anderson**
Updates deps to remove vulnerabilities.

## One known issue.
When updating next I get this warning

```
warn  - ../../node_modules/.pnpm/node-fetch@2.6.7/node_modules/node-fetch/lib/index.js
Module not found: Can't resolve 'encoding' in '/Users/logananderson-forestry-mac/dev/tinacms/tinacms/node_modules/.pnpm/node-fetch@2.6.7/node_modules/node-fetch/lib'
```

It does not seem to effect anything.

### CODE REVIEW

**Comment by jeffsee55** [comment]
> When updating next I get this warning

We should probably make sure we understand why that's happening. Might be related https://github.com/node-fetch/node-fetch/issues/675.

File: package.json [hunk 1]
```diff
@@ -48,5 +48,10 @@
     "cypress": "9.5.3",
     "husky": "^7.0.0",
     "turbo": "^1.3.1"
+  },
+  "pnpm": {
+    "overrides": {
+      "@changesets/assemble-release-plan": "5.1.3"
```
Comment (logan-anderson) [inline package.json:54]: This is needed because of https://github.com/changesets/changesets/issues/910

**Comment by logan-anderson** [comment]
> > When updating next I get this warning
> 
> We should probably make sure we understand why that's happening. Might be related [node-fetch/node-fetch#675](https://github.com/node-fetch/node-fetch/issues/675).

@jeffsee55 I think that is the issue

File: package.json (hunk 1 above)
Comment (logan-anderson) [reply package.json]: Fixed by updating.

**Comment by logan-anderson** [comment]
Some key things to review for the PR
- Local image uploads
- cloudinary image uploads
- General next.js stuff (does the site load, does getStaticProps work, any more warning messages or errors, etc)
- Editing a file, click save and see if the changed file is updated in the file system.

**Comment by logan-anderson** [comment]
> @logan-anderson there are some conflicts with main - you want to resolve before we validate this?

@kldavis4 I have update it. Good catch

**Comment by kldavis4** [comment]
@logan-anderson 

when I run `yarn build` in tina-cloud-starter in the monorepo I get:

```
../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/lib-es5/utils/index.js
Critical dependency: the request of a dependency is an expression

Import trace for requested module:
../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/lib-es5/cloudinary.js
../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/cloudinary.js
../../packages/next-tinacms-cloudinary/dist/handlers.js
./pages/api/cloudinary/[...media].tsx

../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/lib/utils/index.js
Critical dependency: the request of a dependency is an expression

Import trace for requested module:
../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/lib/cloudinary.js
../../node_modules/.pnpm/cloudinary@1.30.0/node_modules/cloudinary/cloudinary.js
../../packages/next-tinacms-cloudinary/dist/handlers.js
./pages/api/cloudinary/[...media].tsx
```

It looks like the build succeeds, so not sure if this is an issue. In the main branch, it fails [completely](https://linear.app/tina/issue/ENG-486/yarn-build-fails-in-tina-cloud-starter-in-monorepo)

**Comment by kldavis4** [comment]
@logan-anderson 
- local image uploads seem to be working normally in the tina-cloud-starter
- cloudinary image uploads don't seem to be currently testable due to: https://linear.app/tina/issue/ENG-485/cloudinary-not-working-in-tina-cloud-starter-in-monorepo
- nextjs stuff seems to be working normally
- editing a file works. Creating a new file wasn't testable due to: https://linear.app/tina/issue/ENG-484/create-document-in-tina-cloud-starter-monorepo-fails
- svgs seem to load as normal in the tina-cloud-starter

**Review by kldavis4 — APPROVED** [review]
Everything in the validation list is now working for me.

@jamespohalloran do we need to make corresponding changes in the standalone tina-cloud-starter repo?

**Comment by jamespohalloran** [comment]
> Everything in the validation list is now working for me.
> 
> @jamespohalloran do we need to make corresponding changes in the standalone tina-cloud-starter repo?

Yep I think all our starters will have to upgrade next, and tinacms once this is out

**Pull request merged**